src/lightning/fabric/plugins/precision/bitsandbytes.py

```diff
@@ -37,7 +39,8 @@
 
 log = logging.getLogger(__name__)
 
-_BITSANDBYTES_AVAILABLE = RequirementCache("bitsandbytes>=0.41.0")
+# TODO: unpin after resolving the `quant_state` format breaking changes
+_BITSANDBYTES_AVAILABLE = RequirementCache("bitsandbytes==0.41.0")
```

Comment:
Still new to this repo, so maybe I misunderstood something.

Not sure how it can work.
In `requirements/pytorch/extra.txt` the BNB is fixed to 0.41.1.
The same goes to tests: `skipif` will always return True. That's why these tests are skipped.

<img width="909" alt="Screenshot 2023-12-27 at 7 02 15 PM" src="https://github.com/Lightning-AI/pytorch-lightning/assets/58434077/5e863def-f82c-44b0-941b-f00343975925">

Reply:
Oh great catch. I messed upthe requirements, I'll open a PR